main_test.go

```diff
@@ -205,7 +205,7 @@ func runSuccess(t *testing.T, args []string, wantUpPath string, wantDownPath str
 	os.Args = args
 
 	if err := run(); err != nil {
-		t.Fatalf("run: args: %s\ngot:  %q\nwant: no error", args, err)
+		t.Fatalf("run: args: %s\nhave: %q\nwant: no error", args, err)
```

Comment:
For my understanding, why `have` is better than `got`?

Reply:
It is a matter of taste at the end. Go code uses both. I have been ambivalent for a lot of time and started using got.
At the end, I convinced myself that have is better for two reasons.
One is that when using separate lines for the output, have aligns naturally with want (both have 4 letters, while if you use got, you need to add a space just to reach the same horizontal space).
The other is that "have" or "got" are also used in some error messages (not only in test code) and I have the impression that "have" reads better.